client/src/pages/Organization/create-event/utils/categories.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react';
+import { useTheme } from '@mui/material/styles';
+import PropTypes from 'prop-types';
+
+import {
+  OutlinedInput, InputLabel, FormControl, MenuItem, Select, Chip, Box,
+} from '@mui/material';
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+
+const categoriesArray = [
+  'Art',
+  'Sport',
+  'Technology',
+  'Formal',
+  'Workshops',
+  'Music',
+  'Food',
+  'Networking',
+  'Business',
+];
```

Comment:
if there is more categories please add it here in a comment

Reply:
literature, politics, social, entertainment, education